Error installing NApps via "kytos napps install" in kytos 2020.2b2

After install kytos 2020.2b2 (using pip or git clone) and trying to install flow_manager or of_core NApps using "kytos napps install kytos/of_core" command the following message is showed:
```
INFO    NApp kytos/of_core:
INFO      Downloading from NApps Server...
ERROR Error installing NApp.
```

And at the controller the following error is showed:
`AssertionError: A name collision occurred between blueprints <flask.blueprints.Blueprint object at 0x7fe2be6e94e0> and <flask.blueprints.Blueprint object at 0x7fe2be6c2b38>. Both share the same name "kytos/of_core". Blueprints that are created on the fly need unique names.`

The complete log error:
```
kytos $> 2020-10-28 11:40:16,863 - WARNING [kytos.core.napps.manager] (Thread-5) NApps dir (/home/jose/temp/kytos/env36/var/lib/kytos/napps/.installed) doesn't exist.
2020-10-28 11:40:16,867 - WARNING [kytos.core.napps.manager] (Thread-6) NApps dir (/home/jose/temp/kytos/env36/var/lib/kytos/napps/.installed) doesn't exist.
2020-10-28 11:40:16,867 - WARNING [kytos.core.napps.manager] (Thread-6) NApps dir (/home/jose/temp/kytos/env36/var/lib/kytos/napps/.installed) doesn't exist.
2020-10-28 11:40:18,029 - INFO [kytos.core.napps.manager] (Thread-6) New NApp installed: kytos/of_core
2020-10-28 11:40:18,136 - ERROR [kytos.core.controller] (Thread-6) Exception on /api/kytos/core/napps/kytos/of_core/install [GET]
Traceback (most recent call last):
  File "/home/jose/temp/kytos/env36/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/jose/temp/kytos/env36/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/jose/temp/kytos/env36/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/jose/temp/kytos/env36/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/jose/temp/kytos/env36/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/jose/temp/kytos/env36/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/jose/temp/kytos/env36/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/jose/temp/kytos/kytos/core/api_server.py", line 498, in _install_napp
    if not self.napps_manager.install(napp, enable=True):
  File "/home/jose/temp/kytos/kytos/core/napps/manager.py", line 69, in install
    return self.enable(napp.username, napp.name)
  File "/home/jose/temp/kytos/kytos/core/napps/manager.py", line 129, in enable
    self._controller.load_napp(username, napp_name)
  File "/home/jose/temp/kytos/kytos/core/controller.py", line 787, in load_napp
    self.api_server.register_napp_endpoints(napp)
  File "/home/jose/temp/kytos/kytos/core/api_server.py", line 352, in register_napp_endpoints
    self.app.register_blueprint(napp_blueprint)
  File "/home/jose/temp/kytos/env36/lib/python3.6/site-packages/flask/app.py", line 98, in wrapper_func
    return f(self, *args, **kwargs)
  File "/home/jose/temp/kytos/env36/lib/python3.6/site-packages/flask/app.py", line 1161, in register_blueprint
    % (blueprint, self.blueprints[blueprint.name], blueprint.name)
AssertionError: A name collision occurred between blueprints <flask.blueprints.Blueprint object at 0x7fe2be6e94e0> and <flask.blueprints.Blueprint object at 0x7fe2be6c2b38>. Both share the same name "kytos/of_core". Blueprints that are created on the fly need unique names.
```

--- a/kytos/core/napps/manager.py
+++ b/kytos/core/napps/manager.py
@@ -125,8 +125,6 @@ class NAppsManager:
             try:
                 # Create symlink
                 enabled.symlink_to(installed)
-                if self._controller is not None:
-                    self._controller.load_napp(username, napp_name)
                 LOG.info("NApp enabled: %s", napp_id)
             except FileExistsError:
                 pass  # OK, NApp was already enabled

